From 6a6aa3012bd1e256fc9d5fff57f63ee8a0eb08f0 Mon Sep 17 00:00:00 2001
From: Giuseppe Scrivano <gscrivan@redhat.com>
Date: Wed, 29 Jan 2014 17:15:17 +0100
Subject: [PATCH] doc: state correctly the number of possible values for
 --network=

95170e8892ecac6a8762990d4cbef9bac14095b2 added --network=direct
without updating the number of possible values for --network in the
man page.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>
(cherry picked from commit 770b173337464301d2970fccaca4128dc514eb1b)
---
 man/en/virt-install.pod    | 2 +-
 man/en/virt-install.pod.in | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/en/virt-install.pod b/man/en/virt-install.pod
index 39ebc6a..811b36c 100644
--- a/man/en/virt-install.pod
+++ b/man/en/virt-install.pod
@@ -620,7 +620,7 @@ This option is deprecated in favor of C<--disk ...,sparse=false,...>
 =item -w NETWORK, --network=NETWORK,opt1=val1,opt2=val2
 
 Connect the guest to the host network. The value for C<NETWORK> can take
-one of 3 formats:
+one of 4 formats:
 
 =over 4
 
diff --git a/man/en/virt-install.pod.in b/man/en/virt-install.pod.in
index 3f6d23e..84edcd2 100644
--- a/man/en/virt-install.pod.in
+++ b/man/en/virt-install.pod.in
@@ -556,7 +556,7 @@ This option is deprecated in favor of C<--disk ...,sparse=false,...>
 =item -w NETWORK, --network=NETWORK,opt1=val1,opt2=val2
 
 Connect the guest to the host network. The value for C<NETWORK> can take
-one of 3 formats:
+one of 4 formats:
 
 =over 4
 
-- 
1.8.5.3

